proxmox/nodes/nodestorage/client.go

```diff
@@ -0,0 +1,41 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+package nodestorage
```

Comment:
Perhaps rename `nodestorage` to just `storage`? Similar to what we have for `node/containers`, `node/tasks`, etc.

Reply:
There is already `storage` package at `proxmox/storage/client.go` for example, for "global" storage api I suppose, this is why i came up with `nodestorage`.

But technically it can be named the same I think? so no problem

Reply:
Yeah, I think `/proxmox/nodes/storage` package will be fine. In in some places we need to import both "storages", when we can use `nodestorage` [import alias](https://stackoverflow.com/a/10408720).